app/models/notification.rb

```diff
@@ -68,7 +68,7 @@ def set_notification_recipients
 
   def set_custom_attributes
     return unless notification_blueprint # let validation catch this.
-    self.actions = UINotifications::URLResolver.new(
+    self.actions ||= UINotifications::URLResolver.new(
```

Comment:
Yes, it's needed. If it's `=` , the blueprint actions:

```ruby
 :actions => {"links"=>[{"title"=>"URL"}]}
```

will override the actual actions created by `UINotifications::RssNotificationsChecker`

```ruby
actions => {
            :links => [
              {
                :href => item.link.href,
                :title => _('Open blog'),
                :external => true
              }
            ]
          }
```